### PR TITLE
Fix many things that were supposed to be removed during build not being removed

### PR DESCRIPTION
**Pull request by eternoendless**
| Questions     | Answers
| ------------- | -------------------------------------------------------
| Branch?       | 1.7.6.x
| Description?  | During build, some files are supposed to be removed. Not all of them were.
| Type?         | improvement
| Category?     | CO
| BC breaks?    | no
| Deprecations? | no
| Fixed ticket? | n/a
| How to test?  | Create a build and see the files

Now being removed:

- Things that were supposed to be removed from `admin`, because the folder was renamed before cleaning it up
- Legacy tests
- .estlintignore, .eslintrc.js, .php_cs.dist, .docker-compose.yml, .webpack, .composer-lock, .package-lock.json...

---
This change is [Reviewable](https://reviewable.io/reviews/prestashop/prestashop/14579)

### CODE REVIEW

File: tools/build/Library/ReleaseCreator.php [hunk 1]
```diff
@@ -129,6 +138,12 @@ class ReleaseCreator
         'tools/build$',
         'tools/foreignkeyGenerator$',
         '.*node_modules.*',
+        '\.eslintignore$',
+        '\.eslintrc\.js$',
+        '\.php_cs\.dist$',
+        '\.docker-compose\.yml$',
+        'tools/assets$',
+        '\.webpack$',
```
Comment (matks) [inline tools/build/Library/ReleaseCreator.php:146]: Sure about this one ?

**Review by PierreRambaud — DISMISSED** [review]

File: tools/build/Library/ReleaseCreator.php (hunk 1 above)
Comment (PierreRambaud) [reply tools/build/Library/ReleaseCreator.php:146]: Yes, it must be removed

**Comment by eternoendless** [comment]
It turns out that removing the `scss` directory makes some images fail to load:

```json
{  
   "stats":{  
      "start":1562774927780,
      "end":1562775021375
   },
   "BO":{  
      "totalLinks":66,
      "totalRequests":3901,
      "passed":[  

      ],
      "failed":[  
         {  
            "page":"http://localhost/prestashop_off/admin197jt4ja1/index.php/sell/catalog/categories?_token=-y31_3bIbjdb5n64hnauN_nawxxKF_seUqqAqVKih4M",
            "URL":"http://localhost/prestashop_off/admin197jt4ja1/themes/new-theme/scss/img/helper-card/category@3x.png?1.7.6.0",
            "status":"404"
         },
         {  
            "page":"http://localhost/prestashop_off/admin197jt4ja1/index.php/sell/customers/?_token=-y31_3bIbjdb5n64hnauN_nawxxKF_seUqqAqVKih4M",
            "URL":"http://localhost/prestashop_off/admin197jt4ja1/themes/new-theme/scss/img/helper-card/customer@3x.png?1.7.6.0",
            "status":"404"
         },
         {  
            "page":"http://localhost/prestashop_off/admin197jt4ja1/index.php/improve/design/themes/?_token=-y31_3bIbjdb5n64hnauN_nawxxKF_seUqqAqVKih4M",
            "URL":"http://localhost/prestashop_off/admin197jt4ja1/themes/new-theme/scss/img/pages/themes/icon_themes.png?1.7.6.0",
            "status":"404"
         },
         {  
            "page":"http://localhost/prestashop_off/admin197jt4ja1/index.php/improve/design/themes/?_token=-y31_3bIbjdb5n64hnauN_nawxxKF_seUqqAqVKih4M",
            "URL":"http://localhost/prestashop_off/admin197jt4ja1/themes/new-theme/scss/img/pages/themes/icon_layouts.png?1.7.6.0",
            "status":"404"
         },
         {  
            "page":"http://localhost/prestashop_off/admin197jt4ja1/index.php/improve/design/cms-pages/?_token=-y31_3bIbjdb5n64hnauN_nawxxKF_seUqqAqVKih4M",
            "URL":"http://localhost/prestashop_off/admin197jt4ja1/themes/new-theme/scss/img/helper-card/content@3x.png?1.7.6.0",
            "status":"404"
         },
         {  
            "page":"http://localhost/prestashop_off/admin197jt4ja1/index.php/configure/shop/seo-urls/?_token=-y31_3bIbjdb5n64hnauN_nawxxKF_seUqqAqVKih4M",
            "URL":"http://localhost/prestashop_off/admin197jt4ja1/themes/new-theme/scss/img/helper-card/seo@3x.png?1.7.6.0",
            "status":"404"
         },
         {  
            "page":"http://localhost/prestashop_off/admin197jt4ja1/index.php/configure/advanced/employees/?_token=-y31_3bIbjdb5n64hnauN_nawxxKF_seUqqAqVKih4M",
            "URL":"http://localhost/prestashop_off/admin197jt4ja1/themes/new-theme/scss/img/helper-card/team@3x.png?1.7.6.0",
            "status":"404"
         }
      ]
   }
}
```

Kudos to @SimonGrn 

It looks like image extraction through webpack isn't working well.

**Comment by eternoendless** [comment]
Merging this for the nightlies

**Comment by matks** [comment]
@eternoendless Shall we create an issue to fix bad images loaded from scss folder ?

**Comment by eternoendless** [comment]
[Done](https://github.com/PrestaShop/PrestaShop/issues/14757).

**Pull request merged**